annotation-web/src/main/java/eu/europeana/annotation/web/service/impl/BaseAnnotationValidator.java

```diff
@@ -63,7 +64,7 @@ protected void validateAgentBody(Body body) throws ParamValidationException, Pro
      * @throws PropertyValidationException
      */
     protected void validateAgentEntityBody(Body body) throws ParamValidationException, PropertyValidationException {
-
+	if(body==null) return;    	
```

Comment:
if the argument is null, that is an error case that cannot be ignored. To avoid NPE, a PropertyValidationException must be thrown

Reply:
Improved.